api/src/main/resources/swagger3codegen/libraries/okhttp-gson/ApiClient.mustache

```diff
@@ -2,7 +2,7 @@
 
 This is an AoU-specific extension of the default Swagger-codegen okhttp API implementation template.
 
-The original template was copied from the 1.0.21 release of swagger-codegen-generators; see https://github.com/swagger-api/swagger-codegen-generators/blob/v1.0.21/src/main/resources/handlebars/Java/libraries/okhttp-gson/ApiClient.mustache
+The original template was copied from the 3.0.21 release of swagger-codegen; see https://github.com/swagger-api/swagger-codegen/blob/v3.0.21/modules/swagger-codegen/src/main/resources/Java/libraries/okhttp-gson/ApiClient.mustache
```

Comment:
I am completely failing to understand why this change works. The current codegen works properly on the 1.0.2 template. AFAICT the swagger codegen version used for swagger3 was not upgraded in the plugin. This is unsettling, but I don't have further theories on this.

Reply:
resolved offline.